src/codeu/chat/database/Schema.java

```diff
@@ -23,15 +27,22 @@ protected void addField(String name, String props) {
   }
 
   /**
+   * Get the fields in the schema.
+   *
+   * @return The fields.
+   */
+  public Map<String, String> getFields() {
+    return fields;
+  }
+
+  /**
    * Create a table with this schema.
    * If the table already exists, does nothing.
    *
    * @param name The name of the table.
    * @param database The database to create the table in.
-   *
-   * @throws SQLException If an SQL error occurs.
    */
-  public void createTable(String name, Database database) throws SQLException {
```

Comment:
Why change the model to swallowing exceptions instead of letting the caller decide how to handle the exceptions?